### PR TITLE
Change font sizes from px to em units

### PR DESCRIPTION
**Pull request by bifodus**

### CODE REVIEW

**Comment by jsdevel** [comment]
:+1:

**Comment by jsdevel** [comment]
Is this just using the same size, but in em?  If so, I don't believe that's what we want.

**Comment by bifodus** [comment]
Yeah, I mentioned in my first email that this alone won't fix the issue
that you're referring to, but it will make it easier to scale the font size
if we make the fonts relative and then adjust the base font size on the
body element (this hasn't been done yet). In the end I'm not sure if
there's anything magical we need to do though.  We just need to add more
font sizes than our designer gave us, or we could try using viewport sizes
(but then we'll probably diverge from what the designer intended).

Bottom line it's still WIP, and I'm open to ideas.

**Comment by jsdevel** [comment]
Any chance of fixing it tonight or tomorrow night?

**Comment by bifodus** [comment]
Yes
